src/app/login/page.js

```diff
@@ -37,25 +40,25 @@ const Login = () => {
         {/* Header */}
         <div className="h4 mb-2 text-center">Sign In</div>
         {/* ALert */}
-        {show ? (
+        {error ? (
```

Comment:
PRAISE: The error handling is so clean. No need for modals or pop ups. I love ternary operators!